pkgs/top-level/all-packages.nix

```diff
@@ -17990,7 +17990,7 @@ in
 
     mod_dnssd = callPackage ../servers/http/apache-modules/mod_dnssd { };
 
-    mod_evasive = callPackage ../servers/http/apache-modules/mod_evasive { };
+    mod_evasive = throw "mod_evasive is not supported on Apache httpd 2.4";
```

Comment:
Please move this into aliases.nix.

Reply:
@SuperSandro2000 I guess I didn't understand you correctly because what I have doesn't work. Can you please explain? Thank you!

Reply:
I think this is not working because it is nested. Not fully sure what is the correct syntax to get this into aliases.

Reply:
@roberth completely random ping, but you seem like the type of person who might know the answer to this :smile:

How to properly remove `apacheHttpdPackages.mod_evasive`? If I add `apacheHttpdPackages.mod_evasive` into `aliases.nix` then every attribute under `apacheHttpdPackages` becomes unusable.

Thanks!

Reply:
Putting attributes like these `aliases.nix` is pointless, because it has the same effect as defining it in their rightful place like you do here. Why would you create complicated indirections if it doesn't achieve anything? And clearly it's complicated because two experienced maintainers didn't figure out how to do it correctly.
Let's keep it simple.

Reply:
Thanks for your feedback @roberth. I always appreciate you taking the time to comment :+1: